pulp_2to3_migration/app/migrations/0006_distribution_trees.py

```diff
@@ -0,0 +1,69 @@
+# Generated by Django 2.2.11 on 2020-03-20 03:46
+
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('pulp_2to3_migration', '0005_pulp2modulemd_pulp2modulemddefaults'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Pulp2Distribution',
+            fields=[
+                ('pulp_id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('pulp_created', models.DateTimeField(auto_now_add=True)),
+                ('pulp_last_updated', models.DateTimeField(auto_now=True, null=True)),
+                ('distribution_id', models.TextField()),
+                ('family', models.TextField()),
+                ('variant', models.TextField()),
+                ('version', models.TextField()),
+                ('arch', models.TextField()),
+                ('pulp2content', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, related_name='distribution_detail_model', to='pulp_2to3_migration.Pulp2Content')),
+            ],
+            options={
+                'default_related_name': 'distribution_detail_model',
+                'unique_together': {('distribution_id', 'family', 'variant', 'version', 'arch', 'pulp2content')},
+            },
+        ),
+        migrations.AlterUniqueTogether(
+            name='pulp2manifest',
+            unique_together=None,
+        ),
+        migrations.RemoveField(
+            model_name='pulp2manifest',
+            name='pulp2content',
+        ),
+        migrations.AlterUniqueTogether(
+            name='pulp2manifestlist',
+            unique_together=None,
+        ),
+        migrations.RemoveField(
+            model_name='pulp2manifestlist',
+            name='pulp2content',
+        ),
+        migrations.AlterUniqueTogether(
+            name='pulp2tag',
+            unique_together=None,
+        ),
+        migrations.RemoveField(
+            model_name='pulp2tag',
+            name='pulp2content',
+        ),
+        migrations.DeleteModel(
+            name='Pulp2Blob',
+        ),
+        migrations.DeleteModel(
+            name='Pulp2Manifest',
+        ),
+        migrations.DeleteModel(
+            name='Pulp2ManifestList',
+        ),
+        migrations.DeleteModel(
+            name='Pulp2Tag',
+        ),
```

Comment:
These are docker related changes.
@dkliban , do you have this file locally /pulp_2to3_migration/app/plugin/docker/pulp_2to3_models.py ? Do you have a docker entry point in the setup.py?

@dralley, did you see the similar docker related migrations in yours?

@ipanova , ^ fyi, in case you are aware what's going on.

Reply:
:fearful: please don't remove docker!

Reply:
I did not have pulp_container installed when I generated the migration originally. Just pushed a new version.